proxysql/Dockerfile.k8s

```diff
@@ -24,7 +24,7 @@ LABEL org.opencontainers.image.license="GPL+"
 LABEL org.opencontainers.image.authors="info@percona.com"
 
 ENV PXC_VERSION 8.0.23-14.1
-ENV PROXYSQL_VERSION 2.3.2-1.1
```

Comment:
we need to use 2.3.2-1.1 in main :)

Reply:
merged main and reverted proxysql version to one required in the main branch